pkg/asset/manifests/proxy.go

```diff
@@ -95,6 +95,7 @@ func (p *Proxy) Generate(dependencies asset.Parents) error {
 // createNoProxy combines user-provided & platform-specific values to create a comma-separated
 // list of unique NO_PROXY values. Platform values are: serviceCIDR, podCIDR, localhost,
 // 127.0.0.1, api.clusterdomain, api-int.clusterdomain, etcd-idx.clusterdomain
+// and 169.254.196.254 the cloud provider instance metadata address.
```

Comment:
So we support AWS, openstack, azure, soon gcp, vSphere, and also no platform `none`...

either include docs links that back us up that the IP is used on cloud for metadata service in functions docs or in the commit messages..

I'm pretty sure on None this IP is not used..

Reply:
`169.254.196.254` is this the IP from where cloud-init metadata comes from?

Reply:
@abhinavdahiya updated with links that show for aws, azure, gcp and openstack that `169.254.169.254` is used for instance metadata.

@praveenkumar nope, should have been `169` vs `196`.